src/app/components/static-svg-statistics/static-svg-statistics.component.ts

```diff
@@ -0,0 +1,60 @@
+import { ChangeDetectionStrategy, Component, Input, NgZone } from '@angular/core';
+import { Mermaid2svgService, Statistics } from '../../services/mermaid2svg.service';
+import { OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-static-svg-statistics',
+  standalone: false,
```

Comment:
standalone mag weggelaten worden, de default value is al 'false'

Reply:
Gedaan.